pkg/pillar/cmd/baseosmgr/baseosmgr.go

```diff
@@ -165,6 +165,7 @@ func handleBaseOsCreate(ctxArg interface{}, key string, configArg interface{}) {
 	for i, ctc := range config.ContentTreeConfigList {
 		cts := &status.ContentTreeStatusList[i]
 		cts.UpdateFromContentTreeConfig(ctc)
+		cts.ObjType = types.BaseOsObj
```

Comment:
We don't need to explicitly set ```Objtype``` in CT status. In ```cts.UpdateFromContentTreeConfig```, we are copying the ```ObjType``` from CT config in CT status.

Reply:
My bad. We removed ```ObjType``` from CT config.